### DISCUSSION TITLE
Integrate a language server

### DISCUSSION THREAD

**Discussion by pointlessone** [zed-industries/zed#6807] [Feature Requests]
How do I integrate a language server?

Say, I have a random language server. How do I integrate it with Zed?

I've found a few examples in the Configuration docs but the look like they are specific to those specific language servers. Can I configure currently unsupported language server? Can I specify which features to use and what language the server serves?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
